rentpath/barney_fife
# [AB] [73630896] Add support for CoffeeLint

This feature finally works. Now the code is ready for some review and cleanup.
#### Testing

(First...)
1. Checkout this branch and launch everything with `foreman start`
   - Note: uncomment the RabbitMQ line in the `Procfile` if you aren't running the IDG instance of Rabbit
2. Install [ngrok](https://ngrok.com/) and open a tunnel to this service with `ngrok 3000`.

(Then...)
1. Create a repository on GitHub and make a first commit on a `master` branch.
2. Make a webhook and set it to deliver Pull Request event payloads to your ngrok URL.
3. Branch off of `master` and commit some CoffeeScript code.
4. Open a pull request from that branch and see what happens.
